### PR TITLE
feat(API)!: move deployment related config to deployment spec

### PR DESCRIPTION
**Pull request by sthaha**
This PR moves all deployment of kepler related specs under `deployment` field. 
I additionally cleans up the unsupported tectonic:ui tags - see https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md for supported tags

### CODE REVIEW

File: bundle/manifests/kepler-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -330,5 +327,5 @@ spec:
   provider:
     name: Kepler Operator Contributors
     url: https://sustainable-computing.io/
-  replaces: kepler-operator.v0.7.2
+  replaces: kepler-operator.v0.0.0-e2e
```
Comment (vimalk78) [inline bundle/manifests/kepler-operator.clusterserviceversion.yaml]: needs change

Comment (sthaha) [inline bundle/manifests/kepler-operator.clusterserviceversion.yaml:330]: This shouldn't be the case

Comment (sthaha) [reply bundle/manifests/kepler-operator.clusterserviceversion.yaml]: Thanks for pointing that out

**Review by vimalk78 — APPROVED** [review]
lgtm

**Pull request merged**